CPC-CHARTER.md

```diff
@@ -237,27 +237,32 @@ CPC Chairperson or Director may serve.
 ## Section 8. Board Representation
 
 There are 3 Board seats earmarked for representation from the
-OpenJS Foundation projects.  The first of these board seats is filled by
-the CPC Director as outlined in Section 7 on [Elections][] and is
-always in place.  
+OpenJS Foundation projects.
 
-The second board seat requires 2 Platinum board members to activate
-and will be filled based on processes defined
+### CPC Director
+
+The first of these board seats is filled by the CPC Director as outlined in Section 7 on [Elections][] and is always in place.
+
+**CPC Director eligibility:** The CPC Director must be a current CPC Member
+
+**CPC Director voting:** CPC Members (Voting and Regular) will vote on candidates
+
+### Community Director
+
+The second board seat will be filled based on processes defined
 by the CPC and approved by the Voting CPC members and then the Board. The
 CPC members will review and either re-confirm or update the
 processes once per year.
 
-For the first year after the formation of the OpenJS Foundation, the
-processes for filling this seat will be as follows:
+**Community Director eligibility:** The Community Director must be an [Active OpenJS Collaborator][] at the time of their selection, as defined in the CPC's Governance document.
 
-* As a constellation member of the OpenJS Foundation, the Node.js project
-  will elect one Board representative from either the Technical Steering Committee (TSC) or
-  Community Committee (CommComm). The representative will be selected according to a process
-  defined by the TSC and CommComm.
+**Community Director voting:** CPC Members (Voting and Regular) will vote on candidates
 
 As described in the [Elections][] section a multiple candidate method
 will be be used for the voting for the first and second board members.
 
+### At-Large Director
```

Comment:
I think this terminology conflicts with our "at-large" projects.

Reply:
This language should probably match to what's in the bylaws, (https://bylaws.openjsf.org, §4.3):
* CPC Director
* At Large Community Director
* Additional Community Director

Reply:
Good call. Robin referenced this idea as there is prior art. I should probably just leave it as tied to Individual Membership until that is all sorted.

Reply:
I hadn't seen Brian's comment when I made my previous comment. The names of the board seats in the By Laws are not great. I think we should determine what they board seats look like through this PR and perhaps add on work with the Individual Membership (Supporter) work and then address the By Laws.

Reply:
@brianwarner suggests we not define the titles as such and refer to them as:
> The Second CPC Director (as defined in §4.3(e) in the [OpenJS Foundation bylaws](https://bylaws.openjsf.org))

Not a bad idea.